### ISSUE TITLE
Tag a release

### ISSUE THREAD

**Issue by dvershinin** [openziti/ngx_http_ziti_module#9]
If this module is anywhere stable, please [tag a release version](https://github.com/openziti/ngx_http_ziti_module/releases).

**Comment by rentallect** [comment]
Yes, we typically apply release tags to everything in the OpenZiti project.  This repo will have tags soon.  This is a brand new repo, and is somewhat experimental in its current state.  The next wave of work will bring in tags.
